### ISSUE TITLE
evaluate.py doesn't output right accuracy and cross entropy

### ISSUE THREAD

**Issue by logiccodes** [googlecodelabs/tensorflow-for-poets-2#6]
Hi, I followed [TensorFlow for Poets 2 codelab](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2/#4) procedure to evaluate the model performance on the test set. And I found that the accuracy output from `evaluate.py` was too low to be true, and didn't match `retrain.py` test set accuracy result.

After some analysing, I've figured out that this is due to the different ground truth labels' order between `evaluate.py` and `retrain.py`. **The permutation within `image_lists.keys()` in [this line](https://github.com/googlecodelabs/tensorflow-for-poets-2/blob/45767a9d62b3b52dae97bf45838fa5701879fe31/scripts/evaluate.py#L60) is always changing**.

We need the ground truth label list in the same order as `retrained_labels.txt`, which is [the same one being used in training](https://github.com/googlecodelabs/tensorflow-for-poets-2/blob/45767a9d62b3b52dae97bf45838fa5701879fe31/scripts/retrain.py#L1139-L1140), just like `label_image.py` does.
One possible solution:
```python
...
from scripts.count_ops import load_graph
from scripts.label_image import load_labels
label_file = "tf_files/retrained_labels.txt"
...
    filenames = []
    labels = load_labels(label_file)
        
    #for label_index, label_name in enumerate(image_lists.keys()):
    for label_index, label_name in enumerate(labels):
      for image_index, image_name in enumerate(image_lists[label_name][category]):
```

**Comment by MarkDaoust** [comment]
Doesn't this change fix the problem?

https://github.com/googlecodelabs/tensorflow-for-poets-2/commit/9e37dcf6bff1b20231337c785bd85aa178199402#diff-05f14b18058604766fb8b5d67f4a37df
